exonum/src/sandbox/consensus/invalid_message.rs

```diff
@@ -42,12 +45,12 @@ fn test_ignore_message_with_incorrect_signature() {
     let sandbox = timestamping_sandbox();
 
     let propose = Propose::new(
-        VALIDATOR_0,
-        HEIGHT_ZERO,
-        ROUND_ONE,
+        SANDBOXED_VALIDATOR_ID,
```

Comment:
As for me, `SANDBOXED_VALIDATOR_ID` looks weird near `ValidatorId(0)`. Why do we need to export it as a constant?